release-tools/Makefile

```diff
@@ -102,7 +102,7 @@ ifeq ($(PULL_COMPONENTS), true)
 	docker pull $(DATASET_OPERATOR_IMAGE)
 else
 	$(call install_local,csi-s3,$(CSI_S3_IMAGE))
-	$(call install_local,csi-driver-nfs,$(CSI_NFS_IMAGE))
+#	$(call install_local,csi-driver-nfs,$(CSI_NFS_IMAGE))
```

Comment:
Why is this commented out? it doesn't build the nfs driver. if I comment it out it fails to build.
```
go: cannot determine module path for source directory /csi-driver-nfs (outside GOPATH, no import comments)
release-tools/build.make:64: recipe for target 'build-nfsplugin' failed
make: *** [build-nfsplugin] Error 1
make: *** [build-containers] Error 2
```